amcl/test/basic_localization_stage.xml

```diff
@@ -1,10 +1,11 @@
+<?xml version="1.0" ?>
 <!-- setting pose: 47.943 21.421 -0.503
-setting pose: 30.329 34.644 3.142 
+setting pose: 30.329 34.644 3.142
```

Comment:
This is the ground truth I assume. Why is it here twice?

Reply:
I don’t know. Keeping it close to the original ros repo